atrium-api/src/types.rs

```diff
@@ -53,72 +53,6 @@ pub trait Collection: fmt::Debug {
     }
 }
 
-/// A record key (`rkey`) used to name and reference an individual record within the same
-/// collection of an atproto repository.
-#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize)]
-pub struct RecordKey(String);
```

Comment:
Document this as "Removed" in the changelog, with a "(moved to `string::RecordKey`)" note.